openmp/libompd/src/ompd-types.h

```diff
@@ -8,8 +8,6 @@ extern "C" {
 #include "omp-tools.h"
 }
 
-#define OMPD_TYPES_VERSION   20170927 /* YYYYMMDD Format */
-
```

Comment:
The reference file at https://github.com/OpenMP/sources/blob/main/include/ompd-types.h has:

```C
#define OMPD_TYPES_VERSION   20180906
```

Reply:
Made the change accordingly

Reply:
thanks!